examples/nextjs-with-typescript/pages/MuxPlayer.tsx

```diff
@@ -166,11 +166,11 @@ const UrlPathRenderer = ({
     ? `?${new URLSearchParams(Object.fromEntries(stateEntries.map(([k, v]) => [k, JSON.stringify(v)]))).toString()}`
     : ''
   const urlStr = `${origin}${pathname}${urlSearchParamsStr}`;
-  const copyToClipboard = () => { 
-    navigator.clipboard?.writeText(urlStr); 
+  const copyToClipboard = () => {
+    navigator.clipboard?.writeText(urlStr);
   };
   return (
-    <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'flex-start' }}>
+    <div className="url-renderer">
```

Comment:
The two big changes in this file are here to swap to a class and put the styles in css.